packages/xstate-vue/package.json

```diff
@@ -57,7 +57,7 @@
   "devDependencies": {
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/vue": "^4.1.0",
-    "@vue/composition-api": "^0.3.4",
+    "@vue/composition-api": "~0.6.5",
```

Comment:
any reason why this is using `~` over `^`?

Reply:
no, changing back. I was initially going to try and provide backward compat, but not anymore